web/pdf_find_bar.js

```diff
@@ -44,182 +45,175 @@ var PDFFindBar = (function PDFFindBarClosure() {
     }
 
     // Add event listeners to the DOM elements.
-    var self = this;
-    this.toggleButton.addEventListener('click', function() {
-      self.toggle();
+    this.toggleButton.addEventListener('click', () => {
+      this.toggle();
     });
 
-    this.findField.addEventListener('input', function() {
-      self.dispatchEvent('');
+    this.findField.addEventListener('input', () => {
+      this.dispatchEvent('');
     });
 
-    this.bar.addEventListener('keydown', function(evt) {
-      switch (evt.keyCode) {
+    this.bar.addEventListener('keydown', (e) => {
+      switch (e.keyCode) {
         case 13: // Enter
-          if (evt.target === self.findField) {
-            self.dispatchEvent('again', evt.shiftKey);
+          if (e.target === this.findField) {
+            this.dispatchEvent('again', e.shiftKey);
           }
           break;
         case 27: // Escape
-          self.close();
+          this.close();
           break;
       }
     });
 
-    this.findPreviousButton.addEventListener('click', function() {
-      self.dispatchEvent('again', true);
+    this.findPreviousButton.addEventListener('click', () => {
+      this.dispatchEvent('again', true);
     });
 
-    this.findNextButton.addEventListener('click', function() {
-      self.dispatchEvent('again', false);
+    this.findNextButton.addEventListener('click', () => {
+      this.dispatchEvent('again', false);
     });
 
-    this.highlightAll.addEventListener('click', function() {
-      self.dispatchEvent('highlightallchange');
+    this.highlightAll.addEventListener('click', () => {
+      this.dispatchEvent('highlightallchange');
     });
 
-    this.caseSensitive.addEventListener('click', function() {
-      self.dispatchEvent('casesensitivitychange');
+    this.caseSensitive.addEventListener('click', () => {
+      this.dispatchEvent('casesensitivitychange');
     });
 
     this.eventBus.on('resize', this._adjustWidth.bind(this));
   }
 
-  PDFFindBar.prototype = {
-    reset: function PDFFindBar_reset() {
-      this.updateUIState();
-    },
-
-    dispatchEvent: function PDFFindBar_dispatchEvent(type, findPrev) {
-      this.eventBus.dispatch('find', {
-        source: this,
-        type: type,
-        query: this.findField.value,
-        caseSensitive: this.caseSensitive.checked,
-        phraseSearch: true,
-        highlightAll: this.highlightAll.checked,
-        findPrevious: findPrev
-      });
-    },
-
-    updateUIState:
-        function PDFFindBar_updateUIState(state, previous, matchCount) {
-      var notFound = false;
-      var findMsg = '';
-      var status = '';
-
-      switch (state) {
-        case FindStates.FIND_FOUND:
-          break;
+  reset() {
+    this.updateUIState();
+  }
 
-        case FindStates.FIND_PENDING:
-          status = 'pending';
-          break;
+  dispatchEvent(type, findPrev) {
+    this.eventBus.dispatch('find', {
+      source: this,
+      type: type,
```

Comment:
Just a small suggestion: If you're using an editor which has support for plugins, I'd highly suggest enabling a linting/ESLint plugin.
Personally, I've found it immensely helpful to be able to get linting results directly in the editor on a file-by-file basis, and that way you can temporarily enable https://github.com/mozilla/pdf.js/blob/master/.eslintrc#L157 to help catch these cases.